containers/python/rpc.py

```diff
@@ -394,13 +394,14 @@ def add_output(self, output):
                      self.string_content_end_position + output_len] = output
         self.string_content_end_position += output_len
 
-    def send(self, socket):
+    def send(self, socket, event_history):
         socket.send("", flags=zmq.SNDMORE)
         socket.send(
             struct.pack("<I", MESSAGE_TYPE_CONTAINER_CONTENT),
             flags=zmq.SNDMORE)
         socket.send(self.msg_id, flags=zmq.SNDMORE)
         socket.send(self.output_buffer[0:self.string_content_end_position])
+        event_history.insert(EVENT_HISTORY_SENT_CONTAINER_CONTENT)
```

Comment:
This missing event history log did not contribute to #232 , but I've corrected the logging issue while addressing the bug.